guildwars2/guild/sync.py

```diff
@@ -130,6 +130,8 @@ def check(m):
             "sync.leader": ctx.author.id,
             "sync.setupdone": True,
             "sync.on": True,
+            "sync.guildrole": False,
+            "sync.name": answer.content,
```

Comment:
Would be better off getting the name when getting the guild during sync instead of storing it, I think